README.md

```diff
@@ -36,15 +38,33 @@ docker run -p 80:8080 \
 cnwan_adaptor
 ```
 
+## Quickstart
+If you want a minimal working setup, the script ```examples/setup_kubecon_demo.sh``` sets everything up a for you in the adaptor and your SD-WAN controller. Before running it, please:
 
+* Install the bash utility ```jq```
+* Edit the file ```examples/policies_definition.json``` with the tunnels, vpns and sites of your deployment. You can add as many polcies and SLAs as you need. In addition, specify the metadata keys and values used by the CNWAN reader.
```

Comment:
I tend to add clickable links into the repo when I reference a file (and I don't use https:// absolute links, since those may change!). It is quite useful to be able to quickly view the file before checking out the repo, or needing to navigate there manually.

Reply:
Added links for the files using GitHub relative paths.